app/components/create_a_job_sidebar_component/create_a_job_sidebar_component.html.haml

```diff
@@ -0,0 +1,18 @@
+.app-step-nav-related.app-step-nav-related--singular
```

Comment:
Why is this file in a folder? I would keep it consistent with the rest for now and maybe refactor all of them at some point if it makes more sense to have views in folders

Reply:
Ah I didn't mean to leave this there. I was messing around with having the html and css in a folder for the component